Gemfile.lock

```diff
@@ -86,7 +86,7 @@ DEPENDENCIES
   webdrivers (~> 3.0)
 
 RUBY VERSION
-   ruby 2.3.5p376
+   ruby 2.5.5p157
```

Comment:
any reason not to go up to 2.6.5?

Reply:
The reason I only went to 2.5 was to stay in sync with the other repos